pkg/plugins/platform/local.go

```diff
@@ -942,6 +960,10 @@ func GetActiveAppRoot(siteName string) (string, error) {
 		if err != nil {
 			return "", fmt.Errorf("error determining the current directory: %s", err)
 		}
+		_, err = CheckForConf(siteDir)
+		if err != nil {
+			return "", fmt.Errorf("Could not find a site in %s. Please specify a site name or change directories: %s", siteDir, err)
+		}
```

Comment:
If no site name is given to ddev rm and the site's .ddev/config.yaml is missing, I have tested and confirmed that it will be caught here and properly error out telling the user to specify a site name.

Reply:
Yeah, please check in the code to make sure that siteName is never empty so there's no way we can construct a data directory of ~/.ddev

Reply:
we check to see if siteName is empty inside of GetActiveAppRoot and handle it there.